source/features/show-whitespace.css

```diff
@@ -16,3 +16,53 @@
 	background-image: url('data:image/svg+xml,%3Csvg preserveAspectRatio="xMinYMid meet" viewBox="0 0 12 24" xmlns="http://www.w3.org/2000/svg"%3E%3Cpath d="M4.5 11C4.5 10.1716 5.17157 9.5 6 9.5C6.82843 9.5 7.5 10.1716 7.5 11C7.5 11.8284 6.82843 12.5 6 12.5C5.17157 12.5 4.5 11.8284 4.5 11Z" fill="rgba(128, 128, 128, 50%25)"/%3E%3C/svg%3E');
 	background-size: 1ch 1.25em;
 }
+
+/* Set a sensible default for user content */
+.markdown-body {
+	--tab-size: 4;
+}
+
+[data-tab-size='1'] {
+	--tab-size: 1;
+}
```

Comment:
99.9% people will either use 2, 4 or 8 and 0% will use this

```suggestion
```

Reply:
Maybe my memory is failing me, but I think we had a conflict with another extension so we added all of these

I could be waaay off, just mentioning it.

Reply:
I think you're referring to the extension that allowed the user to customize the tab size, which we contributed in order to correctly pick up the right tab size in `show-whitespace`

If that's it, then we're good

Reply:
These rules are used to carry the `tab-size` value into `show-whitespace` CSS variable. This is because we can't use `calculated(tab-size)` but only `var(--tab-size)`